### PR TITLE
Include all the datasets

### PR DESCRIPTION
**Pull request by PaulWessel**
Need to show all of them. Kept mdt out of the check script which specifically try 5 and 6 minutes.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -49,6 +49,12 @@ Information about the GMT data server and mirrors can be found on the
         :img-bottom: /_static/GMT_earth_faa_thumbnail.jpg
         :text-align: center
 
+    .. grid-item-card:: IGPP Earth Free-Air Anomaly Error
+        :link: earth-faaerrors
```
Comment (seisman) [inline docs/index.rst:53]:
```suggestion
        :link: earth-faaerror
```

File: docs/index.rst [hunk 2]
```diff
@@ -49,6 +49,12 @@ Information about the GMT data server and mirrors can be found on the
...
+        :link-type: doc
+        :img-bottom: /_static/GMT_earth_faaerrors_thumbnail.jpg
```
Comment (seisman) [inline docs/index.rst:55]: We don't have this image yet.

Comment (PaulWessel) [reply docs/index.rst:61]: I just added what I think are all the missing images and added code to index.rst but getting some errors. Maybe you can see what is missing.

Comment (PaulWessel) [reply docs/index.rst:61]: Duh. Need to add all the earth-edefl.rst etc etc.  Can @Esteban82 help?

**Comment by seisman** [comment]
It seems you added many images in commit e0f75ad32860d4592838a657a1a1bae3b8879816, which doesn't make sense.

**Comment by PaulWessel** [comment]
> It seems you added many images in commit [e0f75ad](https://github.com/GenericMappingTools/remote-datasets/commit/e0f75ad32860d4592838a657a1a1bae3b8879816), which doesn't make sense.

I thought the files in _static was in git so I added them there. Perhaps I dont remember how this docs work.  I can remove if you can advice.

**Comment by Esteban82** [comment]
> Need to show all of them.

I agree. We should page a page for every data set. 
But I don't know if we should show all of them on the main page. For example the gebco and gebcosi. (This would mean that we don't have to make thumbnail for all the images).

**Comment by PaulWessel** [comment]
Soy, family movie time, back in a few hours

**Comment by PaulWessel** [comment]
I guess I am leaning towards including everything, even the gebcosi.  I know we cannot see much different on thumbnails for Earth relief, gebco, synbath, but gebcosi at least differs over Antarctica, no?
I forgot to run the candidate server update so did that earlier today and the scripts now can make all the plots I think.  Might still need work on the makefile (cannot have make docs if there is a directory named docs, so need to change target name, e.g., make html (which is what the docs/Makefile uses).

**Comment by seisman** [comment]
> > It seems you added many images in commit [e0f75ad](https://github.com/GenericMappingTools/remote-datasets/commit/e0f75ad32860d4592838a657a1a1bae3b8879816), which doesn't make sense.
> 
> I thought the files in _static was in git so I added them there. Perhaps I dont remember how this docs work. I can remove if you can advice.

We already have [`docs/_static/GMT_earth_age.png`](https://github.com/GenericMappingTools/remote-datasets/blob/main/docs/_static/GMT_earth_age.png), but in commit e0f75ad32860d4592838a657a1a1bae3b8879816, you added the new `GMT_earth_age.jpg` (PNG vs JPG).

**Comment by Esteban82** [comment]
We also have a figure with the mix of earth day/night images.

**Comment by PaulWessel** [comment]
Sorry, best to approve this one and then fix a smaller set of issues.

File: docs/index.rst (hunk 1 above)
Comment (PaulWessel) [reply docs/index.rst:59]: That I did fix.

**Review by seisman — APPROVED** [review]

**Pull request merged**